spotlight-client/src/charts/useCreateHatchDefs.tsx

```diff
@@ -0,0 +1,60 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2022 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import React, { useState, useEffect } from "react";
+import { PatternLines } from "@vx/pattern";
+import { CategoricalChartRecord, ItemToHighlight } from "./types";
+import { highlightFade } from "./utils";
+import { STATISTIC_THRESHOLD } from "../constants";
+
+export function useCreateHatchDefs(
+  data: CategoricalChartRecord[],
+  highlighted: ItemToHighlight | undefined,
+  type?: "rate"
```

Comment:
Is this used?

Reply:
Hey! Nice to connect with you too! 
This is redundant part, I changed the approach here. Now fixed